src/Xamarin.Android.Build.Tasks/Tasks/Generator.cs

```diff
@@ -126,6 +128,23 @@ protected override string GenerateCommandLineCommands ()
 		{
 			var cmd = GetCommandLineBuilder ();
 
+			if (NamespaceTransforms?.Any () == true) {
```

Comment:
This is probably fine, but feels weird. Would this be just as good?
```suggestion
			if (NamespaceTransforms != null && NamespaceTransforms.Length > 0) {
```

Reply:
Yes, those are equivalent.  I think `Any` and null-check operator is generally considered the modern "best practice".

Reply:
I generally avoid `using System.Linq;`, but there are some cases where `OrderBy()` is ok.

This is fine, though.